NEWS

```diff
@@ -1,3 +1,11 @@
+Flocker 1.3.0 (2015-09-04)
+=====================================
+
+- New Flocker API endpoints to create, list, renew, and delete lease on a dataset.
```

Comment:
The lease API is implemented but the tests aren't quite finished (https://clusterhq.atlassian.net/browse/FLOC-2741) and the API documentation is still private (https://github.com/ClusterHQ/flocker/blob/master/flocker/control/httpapi.py#L872)
So we should remove this and the similar line in the Release Notes.